app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitListFragment.java

```diff
@@ -47,17 +63,29 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         FirestoreRecyclerOptions<Habit> options = new FirestoreRecyclerOptions.Builder<Habit>()
                 .setQuery(_db.collection("Habits"), Habit.class)
                 .build();
-        _habitItemAdapter = new HabitItemAdapter(options, getActivity());
 
+        /* this code stores the habitId of the existing habits in a list, since snapshots
+        weirdly has the habitId 0 for all habits        */
```

Comment:
So I'm a bit confused here... Is the habit 0 for all habits? Do you manually go in and edit the ID of the habit afterward?